nuxeo-core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/action/computation/MakeBlob.java

```diff
@@ -139,18 +178,43 @@ protected String saveInTransientStore(String commandId, String storeName) {
         return getTransientStoreKey(commandId);
     }
 
+    protected String getOutputStream(String commandId) {
+        String outputStream = EXPOSE_BLOB_STREAM;
+        BulkCommand command = Framework.getService(BulkService.class).getCommand(commandId);
+        if (command != null) {
+            if (command.getParam(SORT_PARAMETER) != null) {
+                sort = command.getParam(SORT_PARAMETER);
```

Comment:
sort and zip should be local variable not fields

Reply:
+1